<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rh</class>
 <widget class="QDialog" name="rh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1487</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-30</y>
     <width>1501</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#CFFCFF;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>AJOUTER ET MODIFIER</string>
    </attribute>
    <widget class="QGroupBox" name="ajouter_p">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>621</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#ajouter_p{
background-color:transparent;
border-radius:10px;
border: 1px solid #6568F3;
color:#6568F3;
}</string>
     </property>
     <property name="title">
      <string>FORMULAIRE</string>
     </property>
     <widget class="QTableView" name="tab_produits">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>481</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border:none;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdp1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>370</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_4{
color:white;
background-color:#6568F3;
border-radius:10px;
}
#pushButton_4:hover{
color:white;
background-color:#FF7244;
border-radius:10px;
}</string>
      </property>
      <property name="text">
       <string>MODIFIER</string>
      </property>
     </widget>
     <widget class="QComboBox" name="poste1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Responsable RH</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Poste</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Num√©ro</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="salaire1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="num1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:10px;
border:1px solid #6568F3;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="ajouter_p_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>801</width>
       <height>411</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat ExtraBold</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#ajouter_p_2{
background-color:transparent;
border-radius:10px;
border: 1px solid #6568F3;
color:#6568F3;
}</string>
     </property>
     <property name="title">
      <string>Affichage</string>
     </property>
     <widget class="QTableView" name="tab_produits_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>481</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border:none;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableemploye1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>731</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
